_config.yml

```diff
@@ -202,6 +202,20 @@ gitment:
     # Your client secret:
     client_secret:
 
+# Your Gitalk information
+gitalk:
+    # Switch
+    enable: false
+    # Your Github ID (Github username):
+    github_id:
```

Comment:
I suggest `owner` so we have the same wording as Gitalk and users can check Gitalk documentation to set it up. 

I also suggest we take the description of each option from the Gitalk documentation so it's super explicit.
``` yaml
# Your GitHub Application Client ID
client_id:
# Your GitHub Application Client Secret
client_secret:
# Your GitHub username
owner:
# The GitHub repository which will host comments
repo: 
```

Reply:
Good call. I'll update the wording to reflect that.